cross-chain/arbitrum/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "cross-chain-arbitrum",
+  "version": "0.0.1",
+  "description": "Cross chain Arbitrum",
+  "license": "MIT",
```

Comment:
We use GPL-3.0-only for core `solidity` package:
```suggestion
  "license": "GPL-3.0-only",
```

Reply:
Bumping this up @dimpar @pdyraga

Reply:
Let's use GPL-3.0-only everywhere.